gedditlive/GedditSDK
# Xcode 12 Support

# Changelog

- Added: Xcode 12 support.

# Testing

In order to test using this branch, you should edit the cocoapods repository used as a source for your Podfile:

1. Make sure that `~/.cocoapods/repos/gedditlive` exists
    - If not, Run `pod install` with the following Podfile:

```ruby
source 'https://github.com/gedditlive/GedditSDK.git'
target 'YOUR_TARGET' do
  use_frameworks!

  pod 'GedditCoreSDK', '1.1.3'
  pod 'GedditCommerceSDK', '1.3.1'
end
```

2. Switch to the `release-xcode-fix` branch:

```sh
cd ~/.cocoapods/repos/gedditlive;
git checkout release-xcode-fix
```

3. Edit the podspec files to make `pod install` downloads the frameworks on `release-xcode-fix` branch.

_~/.cocoapods/repos/gedditlive/GedditCoreSDK/1.1.3/GedditCoreSDK.podspec:_
```ruby
s.source = { :http => 'https://github.com/gedditlive/GedditSDK/blob/release-xcode-fix/GedditCoreSDK/1.1.3/GedditCoreSDK.zip?raw=true'}
```

_~/.cocoapods/repos/gedditlive/GedditCommerceSDK/1.3.1/GedditCommerceSDK.podspec:_
```ruby
s.source = { :http => 'https://github.com/gedditlive/GedditSDK/blob/release-xcode-fix/GedditCommerceSDK/1.3.1/GedditCommerceSDK.zip?raw=true'}
```

4. Make sure to clear your pod cache:

```sh
rm -rf ~/Library/Caches/CocoaPods/Pods/Release/GedditCommerceSDK
rm -rf ~/Library/Caches/CocoaPods/Pods/Release/GedditCoreSDK
```

5. Run `pod install` in your project
